proposals/migrate-cli.md

```diff
@@ -0,0 +1,68 @@
+# Migrate Bigchaindb cli for Tendermint
+
+## Problem Description
+With Tendermint integration some of the cli sub-commands have been rendered obsolete. It would be only appropriate to white list sub-commands depending on the backend.
+
+### Use cases
+- Avoid confusing the user by not displaying irrelevant sub-commands.
+
+
+## Proposed Change
+Following sub-commands should be updated
+
+- `bigchaindb --help`: list the relevant sub-commands depending on the configured backend. In case the backend is not configured then the default backend `mongodb` should be assumed.
+
+Following sub-commands should be depreciated for `localmongodb` backend.
+
+- `bigchaindb export-my-pubkey`
+- `bigchaindb set-shards`
+- `bigchaindb set-replicas`
```

Comment:
What about `bigchaindb add-replicas` and `bigchaindb remove-replicas`?

Reply:
I was a little unsure about these as they could different meaning for BDB 2.0 i.e. `add-replicas` could allow the user to setup redundant nodes. The same could be achieved by directly configuring mongodb which in my opinon is a better way.

This is up for discussion.

Reply:
Indeed, "local MongoDB" (with Tendermint) might, in fact, be an entire sharded cluster (for use only by that one BigchainDB node)...